specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transfers.json

```diff
@@ -0,0 +1,1182 @@
+        "StandardAzureSubscription",
+        "AzureReservation",
+        "SavingsPlan"
+      ],
+      "x-ms-enum": {
+        "name": "eligibleProductType",
+        "modelAsString": true
+      }
+    },
+    "TransferProperties": {
+      "description": "Transfer details",
+      "type": "object",
+      "properties": {
+        "expirationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time at which the transfer request expires."
+        },
+        "transferStatus": {
+          "type": "string",
+          "$ref": "#/definitions/TransferStatus",
+          "description": "Overall transfer status.",
+          "readOnly": true
+        },
+        "recipientEmailId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user to whom the transfer request was sent."
+        },
+        "initiatorEmailId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user who sent the transfer request."
+        },
+        "canceledBy": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user who canceled the transfer request."
+        },
+        "detailedTransferStatus": {
+          "type": "array",
+          "readOnly": true,
+          "description": "Detailed transfer status.",
+          "items": {
+            "$ref": "#/definitions/DetailedTransferStatus"
+          }
+        }
+      }
+    },
+    "PartnerTransferProperties": {
+      "description": "Transfer Details.",
+      "type": "object",
+      "properties": {
+        "expirationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time at which the transfer request expires."
+        },
+        "transferStatus": {
+          "type": "string",
+          "$ref": "#/definitions/TransferStatus",
+          "description": "Overall transfer status.",
+          "readOnly": true
+        },
+        "recipientEmailId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user to whom the transfer request was sent."
+        },
+        "initiatorEmailId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user who sent the transfer request."
+        },
+        "resellerId": {
+          "type": "string",
+          "description": "Optional MPN ID of the reseller for transfer requests that are sent from a Microsoft Partner Agreement billing account.",
+          "readOnly": true
+        },
+        "resellerName": {
+          "type": "string",
+          "description": "Optional name of the reseller for transfer requests that are sent from Microsoft Partner Agreement billing account.",
+          "readOnly": true
+        },
+        "canceledBy": {
```

Comment:
Canceled and Cancelled are both valid spellings, but we're using Cancelled in other APIs. Could we change that here?